File: h2hell-web/src/main/java/io/highway/to/urhell/rest/domain/MessageThunderApp.java
package io.highway.to.urhell.rest.domain;

import io.highway.to.urhell.domain.BreakerData;

import java.util.List;

public class MessageThunderApp {
	
	private String token;
	private List<BreakerData> listBreakerData;
	public String getToken() {
		return token;
	}
	public void setToken(String token) {
		this.token = token;
	}
	public List<BreakerData> getListBreakerData() {
		return listBreakerData;
	}
	public void setListBreakerData(List<BreakerData> listBreakerData) {
		this.listBreakerData = listBreakerData;
	}
	

}

File: h2h-PHP-SYMFONY/H2h/domain/MessageThunderApp.php
<?php
namespace H2h\domain;
use \JsonSerializable;
class MessageThunderApp implements JsonSerializable{


	private $token;
	//List<BreakerData>
    private $listBreakerData;

    public function __get($property) {
	    if (property_exists($this, $property)) {
	      return $this->$property;
	    }
  	}

  public function __set($property, $value) {
	  if (property_exists($this, $property)) {
	      $this->$property = $value;
	    }
    return $this;
  	}
  	public function jsonSerialize(){
        return get_object_vars($this);
    }
}